public/director-upload-movie.php

```diff
@@ -25,27 +25,33 @@
             <input type="text" id="MovieTitle" name="MovieTitle" placeholder="Type here your movie title">
           </div>
           <div class="spaceupload">
-            <label for="Category" class="spacetextupload">Category</label>
-            <select name="Category" id="Category" class="styleselect">
-              <option value="null">Choose category</option>
-              <option value="">Action</option>
-              <option value="">Comedy</option>
-              <option value="">Drama</option>
-              <option value="">Documentary</option>
-              <option value="">Fantasy</option>
-              <option value="">Horror</option>
-              <option value="">Mystery</option>
-              <option value="">Romance</option>
-              <option value="">Sci-if</option>
-              <option value="">Thriller</option>
-              <option value="">Other</option>
-             </select>
+          <p class="spacetextupload">Category</p>
+            <div class="multi-selector">
+              <div class="select-field">
+              <input type="text" name="Category" placeholder="Choose Category" id="Category" class="input-selector" disabled>
+              <span class="down-arrow">&blacktriangledown;</span>
+              </div>
+              <div class="list">
+                <label for="Action" class="task"><input type="checkbox" name="Action" id="Action"> Action</label>
+                <label for="Comedy" class="task"><input type="checkbox" name="Comedy" id="Comedy"> Comedy</label>
+                <label for="Drama" class="task"><input type="checkbox" name="Drama" id="Drama"> Drama</label>
+                <label for="Documentary" class="task"><input type="checkbox" name="Documentary" id="Documentary"> Documentary</label>
+                <label for="Fantasy" class="task"><input type="checkbox" name="Fantasy" id="Fantasy"> Fantasy</label>
+                <label for="Horror" class="task"><input type="checkbox" name="Horror" id="Horror"> Horror</label>
+                <label for="Mystery" class="task"><input type="checkbox" name="Mystery" id="Mystery"> Mystery</label>
+                <label for="Romance" class="task"><input type="checkbox" name="Romance" id="Romance"> Romance</label>
+                <label for="Sci-if" class="task"><input type="checkbox" name="Sci-if" id="Sci-if"> Sci-if</label>
+                <label for="Thriller" class="task"><input type="checkbox" name="Thriller" id="Thriller"> Thriller</label>
+                <label for="Western" class="task"><input type="checkbox" name="Western" id="Western"> Western</label>
+                <label for="Other" class="task"><input type="checkbox" name="Other" id="Other"> Other</label>
```

Comment:
Op dit moment is het allemaal een unieke checkbox die je kan afvinken, maar komt het ook binnen als 13 aparte waardes. Zou je willen kijken of je dit ook nog kan verwerken naar een array? bijvoorbeeld door overal name="genre[]" neer te zetten.

`<input type="checkbox" name="genre[]" value="action"/>
<input type="checkbox" name="genre[]" value="comedy"/>
<input type="checkbox" name="genre[]" value="drama"/>
<input type="checkbox" name="genre[]" value="fantasy"/>
`

dit is ook een punt wat wel mist, de value